packages/devextreme/testing/tests/DevExpress.ui.widgets/contextMenu.tests.js

```diff
@@ -273,6 +273,37 @@ QUnit.module('Rendering', moduleConfig, () => {
     });
 });
 
+QUnit.module('Repaint', moduleConfig, () => {
+    QUnit.test('On repaint all submenus should be hidden without console errors (T1257288)', function(assert) {
+        if(!isDeviceDesktop(assert)) {
+            return;
+        }
+
+        const instance = new ContextMenu(this.$element, {
+            items: [{ text: 'Item 1', items: [{ text: 'Item 11' }, { text: 'Item 12' }] }],
+            visible: true,
+            showSubmenuMode: { name: 'onHover', delay: 0 },
+            onItemClick: (e) => e.component.repaint(),
+        });
+
+        const $itemsContainer = instance.itemsContainer();
+        const $rootItem = $itemsContainer.find(`.${DX_MENU_ITEM_CLASS}`).eq(0);
+
+        $($itemsContainer).trigger($.Event('dxhoverstart', { target: $rootItem.get(0) }));
+        this.clock.tick(0);
+
+        const $submenus = $(`.${DX_SUBMENU_CLASS}`);
+        const $nestedSubmenu = $submenus.eq(1);
+        const $nestedSubmenuItem = $nestedSubmenu.find(`.${DX_MENU_ITEM_CLASS}`).eq(0);
+
+        assert.strictEqual($submenus.length, 2, 'Nested submenu is shown');
+
+        $($nestedSubmenuItem).trigger('click');
```

Comment:
```suggestion
        $($nestedSubmenuItem).trigger('dxclick');
```

Reply:
Because when I run tests in batch the test is ever green

Reply:
fixed